net/zerotier/patches/0002-gcc10.patch

```diff
@@ -1,3 +1,12 @@
+From 28929d58f5173346e2d4a80230df4abc5e79726f Mon Sep 17 00:00:00 2001
+From: Moritz Warning <moritzwarning@web.de>
+Date: Tue, 8 Mar 2022 19:10:09 +0100
+Subject: [PATCH 2/5] gcc10
```

Comment:
This patch can be upstreamed, right?

Reply:
same as for the "add cerrno header for (str)errno" patch

Reply:
Any reference for it? I could not see any pull request in ZeroTierOne repository related to this.

Reply:
This change is already upstream. The inserted it further up. I was expecting a merge conflict. :P